addon/evaluate-conditions.js

```diff
@@ -1,4 +1,11 @@
+/**
+ * @module evaluate-conditions
```

Comment:
I just realized this is probably *technically* a breaking change, since I'm renaming a module that could be imported by a consumer. If desired, I can add an `addon/convert-schema.js` that imports and exports the function from `addon/evaluate-conditions.js` and throws a deprecation warning.